docs/manual/_sources/manual_backend.rst

```diff
@@ -533,7 +533,7 @@ Below we show how the :py:class:`CirqStateSampleBackend` from the ``pytket-cirq`
 .. jupyter-execute::
 
     from qiskit.primitives import BackendSampler
-    from qiskit.algorithms import Grover, AmplificationProblem
+    from qiskit_algorithms import Grover, AmplificationProblem
```

Comment:
This is the minor syntax change that is needeed to fix the deprecation warning.

qiskit-algorithms is a separate package